packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/marker.dart

```diff
@@ -140,26 +140,38 @@ class Marker implements MapsObject<Marker> {
   /// * reports [onDragEnd] events
   const Marker({
     required this.markerId,
+    this.icon,
     this.alpha = 1.0,
     this.anchor = const Offset(0.5, 1.0),
     this.consumeTapEvents = false,
     this.draggable = false,
     this.flat = false,
-    this.icon = BitmapDescriptor.defaultMarker,
     this.infoWindow = InfoWindow.noText,
     this.position = const LatLng(0.0, 0.0),
     this.rotation = 0.0,
     this.visible = true,
     this.zIndex = 0.0,
     this.onTap,
+    this.count,
+    this.price,
     this.onDrag,
     this.onDragStart,
     this.onDragEnd,
-  }) : assert(alpha == null || (0.0 <= alpha && alpha <= 1.0));
+  }) : assert(0.0 <= alpha && alpha <= 1.0 && (price != null || count != null));
 
   /// Uniquely identifies a [Marker].
   final MarkerId markerId;
 
+  /// The number that will be placed at the cluster marker, the number of
+  /// houses in the region.
+  final int? count;
+
+  /// The price of the house represented by the price marker.
+  final String? price;
```

Comment:
What about mentioning in both comments that if `count` is set, then `price` should not and vice versa.

Reply:
That would be better. Will do!